### PR TITLE
Yechezkel/db_sql1

### PR DESCRIPTION
**Pull request by chezi-nikop**
sql class ex is done

### CODE REVIEW

File: src/main/resources/db/migration/V2024008291400__create_pets_table.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+CREATE TABLE Pets
+(
+    id SERIAL primary key,
+    name varchar(255),
```
Comment (mayhuberman) [inline src/main/resources/db/migration/V2024008291400__create_pets_table.sql:4]: pay attention, what columns should hold non null values?

Comment (mayhuberman) [reply src/main/resources/db/migration/V2024008291400__create_pets_table.sql:4]: and what should have default values?

File: src/main/resources/db/migration/V2024008291457__create_owner_table.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE Owner
+(
+    id SERIAL primary key,
+    name varchar(255),
+    type varchar(255),
```
Comment (mayhuberman) [inline src/main/resources/db/migration/V2024008291457__create_owner_table.sql:5]: what is type in owner table?

**Comment by chezi-nikop** [comment]
fixed all prob, can you please check it?

**Review by omermelamed — APPROVED** [review]

File: src/main/resources/db/migration/V202409091400__create_pets_table.sql [hunk 3]
```diff
@@ -0,0 +1,30 @@
+create table Pets
```
Comment (mayhuberman) [inline src/main/resources/db/migration/V202409091400__create_pets_table.sql:1]: table name should start with a small letter

File: src/main/resources/db/migration/V202409091457__create_owner_table.sql [hunk 4]
```diff
@@ -0,0 +1,10 @@
+create table Owner
```
Comment (mayhuberman) [inline src/main/resources/db/migration/V202409091457__create_owner_table.sql:1]: same here

**Review by mayhuberman — APPROVED** [review]

**Pull request merged**